Source/UI/NewUI/PaymentForm/View/CardNumber/CardNumberView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CardNumberView.swift
+//  Frames
+//
+//  Created by Harry Brown on 05/07/2022.
+//  Copyright © 2022 Checkout. All rights reserved.
+//
+
+import UIKit
+
+protocol CardNumberViewProtocol: AnyObject {
+  var schemeIcon: Constants.Bundle.SchemeIcon { get set }
```

Comment:
When do we envision to reuse CardNumberViewProtocol such that we benefit now from creating the interface?

Reply:
We use this for tests